### ISSUE TITLE
[macOS] Minimize to Tray/Command-Q not working

### ISSUE THREAD

**Issue by rdela** [keybase/client#23012]
Version 5.3.0-20200310172631+4f2689009b
macOS 10.14.6 (18G3020)

"Minimize to Tray" (Command-Q) is not working for me. Command-Q makes Keybase flash in the menu bar but the window remains in view. Clicking the menu item (Keybase > Minimize to Tray Command-Q) also has no effect.

Clicking the red stoplight window menu works, as does Command-W.

"Quit Keybase" in the menubar hamburger menu and "Quit" from control-clicking on the dock icon both work and quit the app completely, including the menubar app.

Related but not identical issues 
https://github.com/keybase/client/issues/4721
[…]
https://github.com/keybase/client/issues/20580

Notably, there are tray issues on other platforms as well, possibly shifting the balance to make Command-Q and its equivalents on other platforms all behave like "Quit Keybase" in the menubar hamburger menu and "Quit" from control-clicking on the dock icon. For example:
https://github.com/keybase/client/issues/4715
https://github.com/keybase/client/issues/5824
https://github.com/keybase/client/issues/6447
https://github.com/keybase/client/issues/22930

**Comment by DaveWoodCom** [comment]
Came here to file the same bug, just adding on to this one instead of creating a new issue.

On macOS Catalina, 'Minimize to Tray' no longer works in the new Keybase (5.3.0). I confirmed on a second machine that still had 5.2.x on it, that it
worked there, then upgraded it to 5.3, and it stopped working. So something broke between those two versions.

my log id: 8bcbc38506d7a693b3ee9b1c

**Comment by Ambros94** [comment]
Same issue on MacOS 10.15.3 and KeyBase yersion 5.3.0-20200310172631+4f2689009b

**Comment by RecuencoJones** [comment]
Same, MacOS 10.14.6, Keybase version 5.3.0-20200310172631+4f2689009b

**Comment by tommythorn** [comment]
I was halfway filing

"I love Keybase, really, but it's annoying that the client on macOS doesn't act properly like other apps.  It doesn't hide on Cmd-H, doesn't minimize on Cmd-M, and doesn't close on Cmd-W.  There may be other similar issues."

when I noticed this had partially been filed.

**Comment by storyjesse** [comment]
Try Icon vanished after upgrade on Linux Mint 19 Tara Cinnamon 4.0.10 too
keybase version 5.3.1-20200320154633+3e235215b3
I notice that keybase seems to be run with the `service` option, I'm not sure what this means or whether it's related. keybase may have been running like this for months.
```
$ run_keybase %u
Shutting down Keybase GUI...
Unmounting /run/user/1000/keybase/kbfs...
Unmounting and shutting down kbfsfuse...
Shutting down keybase service...
Starting via systemd...
run_keybase: Success!

$ pgrep -a keybase
7682 /usr/bin/keybase-redirector /keybase
8730 /usr/bin/keybase --use-default-log-file --debug service
```

**Comment by rdela** [comment]
Fixed in 5.4.0. "Minimize to Tray" (Command-Q) and clicking the menu item (Keybase > Minimize to Tray Command-Q) both close the window now.

Also a new command "Quit Keybase Completely" added in Keybase menu and bound to Alt/Option-Command-Q (⌥⌘Q). It behaves like "Quit Keybase" in the menubar hamburger menu and "Quit" from control-clicking on the dock icon.

Probably as close to the the ideal solution of making Command-Q behave normally (instead of like Command-W) as we are likely to get, so I am closing this. Command-H does hide the app, and Command-M does minimize the app.
